frontend/src/app/employeeshowresolver.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@angular/core';
+
+import { Resolve } from '@angular/router';
+
+import { Observable } from 'rxjs';
+import { ActivatedRouteSnapshot, RouterStateSnapshot } from '@angular/router';
+import { User } from './models/User.model';
+import { ApiCommunicationService } from './services/ApiCommunication.service';
+
+
+@Injectable()
+export class employeeshowresolver implements Resolve<User>{
```

Comment:
```suggestion
export class EmployeeShowResolver implements Resolve<User>{
```
change everything (including the file name) accordingly